pkg/templates/images/images.go

```diff
@@ -57,6 +57,7 @@ const (
 	HubbleUI
 	HubbleUIBackend
 	HubbleProxy
+	CertGen
```

Comment:
Nit: can you please rename this to CiliumCertGen, so that we know it's a Cilium-related image?